finda-auth/src/main/proto/volunteer.proto

```
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package volunteer;
```

Buf (1.67.0)

[error] Files with package "volunteer" must be within a directory "volunteer" relative to root but were in directory "finda-auth/src/main/proto".

(PACKAGE_DIRECTORY_MATCH)